cli/dcoscli/node/main.py

```diff
@@ -482,15 +484,71 @@ def _log(follow, lines, leader, slave):
 
     if lines is None:
         lines = 10
+
     lines = util.parse_int(lines)
 
-    mesos_files = _mesos_files(leader, slave)
+    try:
+        _dcos_log(follow, lines, leader, slave)
+        return 0
+    except DCOSException as e:
```

Comment:
the reason for try catch is to be able to fall back and try to use files APIs if something goes wrong in dcos_log. Let me know if this is not a desirable behavior and i fix it.